offchain/rollups-events/src/broker/mod.rs

```diff
@@ -273,6 +337,11 @@ pub struct BrokerCLIConfig {
     #[arg(long, env, default_value = "redis://127.0.0.1:6379")]
     redis_endpoint: String,
 
+    /// Address list of Redis cluster nodes, defined by a single string
+    /// separated by empty spaces. If present, it supersedes `redis_endpoint`
+    #[arg(long, env, num_args = 1.., value_delimiter = ' ')]
```

Comment:
Wouldn't it be better to use `,` as the delimiter?

Reply:
Arguably, yes. I went with empty spaces to keep it similar to the `redis-cli` interface, but I have no objections to changing it.